pontoon/insights/tasks.py

```diff
@@ -500,14 +513,18 @@ def build_activity_charts_data(start_of_today, sync_user):
             else:
                 data["peer_approved"].add(translation)
                 if action["approved_date"]:
-                    data["times_to_review"].append(
-                        action["approved_date"] - action["date"]
-                    )
+                    review_time = action["approved_date"] - action["date"]
+                    data["times_to_review_suggestions"].append(review_time)
```

Comment:
Would it be more useful to not include the pretranslation suggestions in this data, going forward?

Reply:
I'd like to say no, but I also think it would be useful to see time to review "human" suggestions only. Maybe there can be a filter for that on the Time to review suggestions chart.

@flodolo ?

Reply:
My gut reaction was that we don't want to differentiate, because a review is a review, independently from the source of the suggestion.

On second thoughts though, it might make sense to have one chart only dedicated to time-to-review for standard suggestions, given we have the equivalent for pretranslations. One chart with a selector would work (not sure how you'd call non-pretranslations though, standard suggestions?)

Reply:
@mathjazz Are you considering applying further changes within this PR, or as potential follow-up work?

Reply:
No in this PR. We should file an issue, though.